### ISSUE TITLE
Elevated plasma 3-OH-Linoleylcarnitine, C18:1-OH

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5984]
**Preferred term label:** Elevated plasma 3-OH-Linoleylcarnitine, C18:1-OH

**Synonyms**

**Definition (free text, please give PubMed ID)** Elevated 3-OH-Linoleylcarnitine concentration in the blood.

**Parent term (use hpo.jax.org/app)** Abnormal circulating acetylcarnitine concentration HP:0012071

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding term
